app/objects/c_link.py

```diff
@@ -35,11 +35,12 @@ def output(self):
         except Exception as e:
             return None
 
-    def __init__(self, operation, command, paw, ability, status=-3, score=0, jitter=0, cleanup=0):
+    def __init__(self, operation, command, paw, ability, agent_name=None, status=-3, score=0, jitter=0, cleanup=0):
```

Comment:
when you create an agent (via agent_svc) I don't see it using the agent_name... talk me through that field

Reply:
This is basically here to preserve the human friendly name on Link objects so they can be displayed instead of Agent id in the web app and when debugging.  We could get rid of this but we'd either need to be happy just having the agent id field displayed in the gui or handle de-refrencing the agent display name either in the front-end or maybe as a property field on Link object. 

Leaving the human friendly name on Links seemed like the simplest / most minimally invasive change to the code base while retaining the same kind of usability.  But, I agree that it might be nicer to have it more normalized.

Reply:
I've dropped the commit that added the Link.agent_name field.  I think that this PR should be pretty uncontroversial now.  I've also updated the chain PR https://github.com/mitre/chain/pull/211/files to only display agent IDs for links (we'll probably want to add something later to dereference the agent display name for links)